pkg/pillar/devicenetwork/dnc.go

```diff
@@ -755,6 +746,25 @@ func checkAndUpdateWireless(ctx *DeviceNetworkContext, oCfg *types.DevicePortCon
 				log.Infof("checkAndUpdateWireless: updated wpa file ok %v\n", status)
 			}
 		}
+
+		// hack, save the override wifi param into ctx for later copying
```

Comment:
// XXX remove when controller support WiFi
would make it easier to find, especially if you use the same comment string in all places where removal is needed.